.hooks/pre-commit

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash -e
+
+echo "Running ${PWD}/$0"
```

Comment:
I assume this is similar to what we have in floodlight...

Reply:
*identical